www/scripts/main.js

```diff
@@ -229,6 +230,11 @@ function setLastRoomName(roomName) {
 
 /* Helpers */
 
+function displayMessage(recipient) {
+    var currentUserName = userNameField.val();
+    return (typeof recipient === 'undefined' || recipient == '' || recipient.toUpperCase() == currentUserName.toUpperCase());
```

Comment:
Also should this show messages you whispered to someone. Otherwise it'll look like it was never sent